packages/docusaurus-plugin-content-docs/src/props.ts

```diff
@@ -45,13 +45,16 @@ Available document ids are:
     const {
       title,
       permalink,
-      frontMatter: {sidebar_label: sidebarLabel},
+      frontMatter: {
+        sidebar_label: sidebarLabel,
+        sidebar_class_name: sidebarClassName,
```

Comment:
sidebar_class_name intuitively implies to me that it will apply as well to doc items in the sidebar, and not just category

Maybe we should make it more explicit?

`sidebar_category_class_name` ? Or apply to doc and categories both?